tools/compiler/src/processor.rs

```diff
@@ -124,6 +126,12 @@ pub fn transport_metadata_along_channel(
     // Without this, decoding will fail because the asset ID won't match.
     pb_metadata.penumbra_asset_id = None;
 
+    // Add priority score
+    pb_metadata.priority_score = priority_scores_by_base
+        .get(&pb_metadata.base)
+        .copied()
+        .unwrap_or(1);
```

Comment:
issue: if it is not in the HashMap, don't think we should add any score to the metadata object. Aka:


```rust
    // Add priority score if available
    if let Some(score) = priority_scores_by_base.get(&pb_metadata.base) {
        pb_metadata.priority_score = *score;
    }
```